src/app/issuer/components/issuer-create/issuer-create.component.html

```diff
@@ -132,18 +134,22 @@ <h2>{{ 'General.address' | translate }}</h2>
 		</label>
 
 		<div class="l-flex l-flex-1x u-margin-bottom3x u-margin-top4x">
```

Comment:
replace classes with	
```
<div class="tw-flex tw-justify-end tw-gap-2 tw-mt-4"> 
```
to make it consistent with other pages.

Reply:
updated.